config/tests/jobs/jobs_test.go

```diff
@@ -287,6 +287,408 @@ func TestRetestMatchJobsName(t *testing.T) {
 	}
 }
 
+// Enforce that we stop adding jobs scheduled to deprecated google-internal CI clusters
+// https://groups.google.com/a/kubernetes.io/g/dev/c/p6PAML90ZOU
+func TestCommunityJobs(t *testing.T) {
+	const legacyDefault = "default"
+	const legacyTrusted = "test-infra-trusted"
+
+	// TODO: cut this list down to none
+	// this is a snapshot of docs/job-migration-todo.dm
+	// we are not overcomplicating this as all of them will be removed in august anyhow
+	// tail -n +10 docs/job-migration-todo.md | cut -d \| -f 3 | awk '{ print "\""$0"\","}'
+	knownBadJobs := sets.NewString(
+		"pull-csi-driver-nfs-e2e",
```

Comment:
locally tested that this is WAI by commenting some of these out temporarily

Reply:
✅